addons/sale/models/res_partner.py

```diff
@@ -55,13 +55,15 @@ def action_view_sale_order(self):
         action["domain"] = [("partner_id", "in", all_child.ids)]
         return action
 
-    def _credit_debit_get(self):
-        super()._credit_debit_get()
+    def _compute_credit_to_invoice(self):
+        # OVERRIDE
```

Comment:
```suggestion
        # EXTENDS 'account'
```
If you call `super()` it's not an override :wink: